outreachy_proposal.md

```diff
@@ -0,0 +1,132 @@
+Also, through voting, the best translation is chosen collaboratively. 
+
+3. Budget planning
+This would enable applicants to manage resources effectively, they can also collaborate with other applicants to avoid excessive budgeting. 
+
+4.  Comments and Issues
+Maintainers, reviewers will be able to ask questions, this will be in form of a Pull request comment on the file/repository on CHAOSS Badging. They will also get feedback based on each activity and request for more context if need be.  
+
+## Approach
+
+<!-- This section needs to be redrafted -->
+
+Technologies 
+- Languages: YAML 
+- Framework: Crowdin
+- Platform: Github
+- Markup: Markdown for writing documentation 
+     
+
+
+|Period | Task  |
+|-------| ------------------------------------|
+|        After submission [ April 7 -  May 8]          |-  More contributions to CHAOSS Badging, - Gather more contents about Badging 
+|       [MAY 8 - MAY 31]         | - Discuss with mentors on any changes or feedback -Gather more content about Crowdin workflow - Create framework for guidelines 
+|       [JUNE 1 - JUNE 14]        | -Meet mentors for feedback and act accordingly -Gather repositories for translation -Get more content, guidelines and procedures that needs to be translated 
+|      [ JUNE 15 - JUNE 30]       | -Meet mentors for feedback and make changes -Gather more knowledge about multilingual languages that will be implemented
+|        [JULY 1 - JULY 5]       |  Phase one Evaluation
+|        [JULY 6 - JULY 28]     | Automate workflow on Github -Implement translators -Integrate the Crowdin workflow on github processes -Synchronize schedule configuration -Gather feedback from mentors and act accordingly
+|         [JULY 29 - JULY 31]       | - Phase two evaluation
+|         [AUGUST 1 - AUGUST 16]        | - Start Documenting the workflow -Add/scale more repositories -Fix any issues that might come up 
+|           [AUGUST 17 - AUGUST 25]     | Wrap up Documentation -Ask mentors for feedback
+|           [ AUGUST 26 - AUGUST 31]     | Submit final work report
+
+
+### About me
+I am Ore-aruwaji Tola currently in my final year studying Electrical & Information Engineering.  I am a  staunch advocate of Open source technologies. I have been contributing to the CHAOSS D&I Badging project for over 2 months. I have experience in AWS, Devops, python and Linux. 
+
+```
+Contributions to CHAOSS D&I Badging
+ https://github.com/badging/diversity-and-inclusion/pull/2
+https://github.com/badging/meta/issues/6
+https://github.com/thecraftman/CHAOSS-microtasks/blob/master/Microtasks_solution.md
+https://github.com/thecraftman/CHAOSS-microtasks
+https://github.com/thecraftman/CHAOSS-microtasks/blob/master/Draft_PR_Event_Badge.md
+
+My other Open Source contributions
+
+https://github.com/thecraftman/NAT-Gateway
+https://github.com/thecraftman/Python-REST-APIs
+https://github.com/thecraftman/Tic-Tac-Toe---Python-
+```
```

Comment:
```suggestions
#### Contributions to the CHAOSS D&I Badging program

- https://github.com/badging/diversity-and-inclusion/pull/2
- https://github.com/badging/meta/issues/6
- https://github.com/thecraftman/CHAOSS-microtasks/blob/master/Microtasks_solution.md
- https://github.com/thecraftman/CHAOSS-microtasks
- https://github.com/thecraftman/CHAOSS-microtasks/blob/master/Draft_PR_Event_Badge.md

#### Contributions to Open Source

- https://github.com/thecraftman/NAT-Gateway
- https://github.com/thecraftman/Python-REST-APIs
- https://github.com/thecraftman/Tic-Tac-Toe---Python-
```

- Markdown Links should only be fenced when they are not intended as hyperlinks.